checkbox-support/checkbox_support/parsers/tests/test_udevadm.py

```diff
@@ -355,9 +355,9 @@ def test_TOSHIBA_NVME(self):
         self.assertEqual(self.count(devices, "VIDEO"), 2)
         self.assertEqual(self.count(devices, "AUDIO"), 2)
         self.assertEqual(self.count(devices, "KEYBOARD"), 1)
-        self.assertEqual(self.count(devices, "TOUCHPAD"), 1)
+        self.assertEqual(self.count(devices, "TOUCHPAD"), 2)
         self.assertEqual(self.count(devices, "CARDREADER"), 1)
-        self.assertEqual(self.count(devices, "MOUSE"), 1)
+        self.assertEqual(self.count(devices, "MOUSE"), 0)
```

Comment:
This doesn't seem right.

I went and check what device this udev DB came from, and it is a [Dell Latitude 5580](https://www.pcmag.com/reviews/dell-latitude-5580) which comes with a trackpoint, similar to the ones on the Lenovo laptops.

I asked one of our colleagues and they provided a sample of a Lenovo unit with such a trackpoint:

```
4649 P: /devices/pci0000:00/0000:00:1f.4/i2c-0/0-0015/input/input8
4650 L: 0
4651 E: DEVPATH=/devices/pci0000:00/0000:00:1f.4/i2c-0/0-0015/input/input8
4652 E: SUBSYSTEM=input
4653 E: PRODUCT=18/4f3/67/0
4654 E: NAME="Elan TrackPoint"
4655 E: PROP=21
4656 E: EV=7
4657 E: KEY=70000 0 0 0 0
4658 E: REL=3
4659 E: MODALIAS=input:b0018v04F3p0067e0000-e0,1,2,k110,111,112,r0,1,amlsfw
4660 E: USEC_INITIALIZED=3293319
4661 E: ID_INPUT=1
4662 E: ID_INPUT_POINTINGSTICK=1
4663 E: ID_INPUT_MOUSE=1
4664 E: ID_SERIAL=noserial
4665 E: ID_PATH=pci-0000:00:1f.4
4666 E: ID_PATH_TAG=pci-0000_00_1f_4
4667 E: ID_FOR_SEAT=input-pci-0000_00_1f_4
4668 E: TAGS=:seat:
```

And, as confirmed with them, in this case, when ID_INPUT_POINTINGSTICK=1 and ID_INPUT_MOUSE=1, the TrackPoint should be a mouse, not a touchpad. (otherwise the touchpad test cases would be run twice when running Checkbox on this device, which doesn't really make sense)

Reply:
The question is, why should a trackpoint be a mouse?

Reply:
Is it possible to add a separate test case for `pointingsitck`? If that's present then it's okay to have no touchpad?